### ISSUE TITLE
Error: glob pattern string required

### ISSUE THREAD

**Issue by cannap** [wallabyjs/quokka#78]
### Issue description or question

When i open a new file with Quoakka i got 
```
[Extension Host]  [Info]  #1 Quokka.js started
25/C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:28 glob pattern string required
```
everytime i hit a key the error popups again
https://i.gyazo.com/c366097e99e763b5a577741850152726.mp4
and when i save the file quoakka just stops working 
### Quokka.js file

```javascript
var fetch = require('node-fetch')
```

### Quokka.js Console Output
```
​​​​Quokka #1 (node: v8.0.0, babel: v6.26.0)​​​​

```

### Code editor version
Visual Studio Code v1.15.1

### OS name and version

Windows
### Node version
v.8.0.0
//config

```json
{
  "pro": true,
  "babel": {
    "path": "C:\\Users\\marko\\AppData\\Roaming\\nvm\\v8.0.0\\node_modules\\babel-core"
  }
}
```

installed extensions

```
2gua.rainbow-brackets
alefragnani.project-manager
chenxsan.vscode-standardjs
christian-kohler.npm-intellisense
codezombiech.gitignore
Compulim.compulim-vscode-closetag
cssho.vscode-svgviewer
esbenp.prettier-vscode
formulahendry.auto-rename-tag
IBM.output-colorizer
lamartire.git-indicators
mohsen1.prettify-json
msjsdiag.debugger-for-chrome
naumovs.color-highlight
nwallace.peep
octref.vetur
patrys.vscode-code-outline
pawelgrzybek.bimbo-theme
PKief.material-icon-theme
robinbentley.sass-indented
romanresh.testcafe-test-runner
ronnidc.nunjucks
Shan.code-settings-sync
sleistner.vscode-fileutils
tunnckocore.modern-javascript-snippets
vincaslt.highlight-matching-tag
WallabyJs.quokka-vscode
wix.vscode-import-cost
xabikos.JavaScriptSnippets
```

thanks for this awesome work!

**Comment by ArtemGovorov** [comment]
Thanks for providing the video. 

After our code review, I didn't find any possible ways how Quokka could produce the exception, so it is possible that it's not Quokka but some other extension that you have installed may be causing the issue.

Try this:
- download and install [VS Code insiders build](https://code.visualstudio.com/insiders),
- start the VS Code insiders build, and install only Quokka extension (and no other extensions) and try running Quokka to see if the error is reproduced.

If it's not reproduced, it means that some other extension is causing the issue. In this case try disabling your main VS Code installation extensions one by one to find the one causing the issue.

Either way, let us know the results of following the above suggested steps.

> and when i save the file quoakka just stops working

If you mean when you create a new Quokka file (with `Ctrl + K, J`), and then save a file in VS Code, then it's expected, because for VS Code it is essentially becomes a different file (tmp -> real file), so you need to start Quokka on the saved file (`Ctrl + K, Q`).
Generally, to (re)start Quokka on an existing file you may use the `Ctrl + K, Q` shortcut (`Start Quokka on Current file` command).

It should however work normally if you start Quokka on an existing file and save the file after that.

**Comment by ArtemGovorov** [comment]
After searching your other extensions source code, I have found out that only `Color Highlight` extension is throwing an error with such message `glob pattern string required`. Try disabling the extension to see how it goes.

**Comment by cannap** [comment]
Nope i found the extension which makes the problem 

chenxsan.vscode-standardjs 

https://marketplace.visualstudio.com/items?itemName=chenxsan.vscode-standardjs 

i tested 1000 times when i deactivate restart vscode it works without error but still stops after saving the file


When i open Quaokka installed with Javascript Standard then Quoakka uses both or something 
you can see on the Bottom
https://gyazo.com/8ac5ab75611a8fd1ef8b35f549d7a625

**Comment by ArtemGovorov** [comment]
Thanks! I can confirm that the `chenxsan.vscode-standardjs` is causing the issue. The [issue had been already reported in the extension repo](https://github.com/chenxsan/vscode-standardjs/issues/37). I have left a comment and the link to this issue there.

For now, if want to use both extensions and not disable the `chenxsan.vscode-standardjs`, you may:
- save a file temporarily somewhere, the run quokka on it (with `Ctrl + K, Q`).
- or change the `standard.run` setting of the `chenxsan.vscode-standardjs` extensions to not run on typing, but only on saving, like [described in the extension docs](https://marketplace.visualstudio.com/items?itemName=chenxsan.vscode-standardjs).

Hope it helps.

**Comment by chenxsan** [comment]
@cannap Sorry about the inconvenience vscode-standardjs brought, I just published 1.1.8 to fix the bug, please update to see if it helps you.

**Comment by cannap** [comment]
works
